packages/components/src/atoms/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { forwardRef } from 'react'
+import type { HTMLAttributes, PropsWithChildren } from 'react'
+
+interface Size {
+  width: string
+  height: string
+}
+
+export type BorderStyle = 'regular' | 'pill' | 'circle'
+
+export interface SkeletonProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+  /**
+   * Control whether skeleton should be visible or not.
+   */
+  loading?: boolean
+  /**
+   * Control whether the shimmer effect should be displayed or not.
+   */
+  shimmer?: boolean
+  /**
+   * Specifies the skeleton element size (width, height).
+   */
+  size: Size
+  /**
+   * Specifies the skeleton element border radius ('regular' | 'pill' | 'circle').
+   */
+  border?: BorderStyle
+  /**
+   * Customizes the skeleton element border size.
+   */
+  customBorder?: string
```

Comment:
Can we change the name of this prop? I think `customBorder` might confuse people, since we're only really using this value to allow them to change the `border-radius` property.

```suggestion
  /**
   * Custom border radius for skeleton elements.
   */
  borderRadius?: string
```

Reply:
nicee! Agreed! :)

Reply:
updated [here](https://github.com/vtex/faststore/pull/1616/commits/d2fe7ea6fbad565fa7d72058c3d8825e8a8ae7f4)